extensions/nanoarrow_ipc/subprojects/nanoarrow.wrap

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[wrap-git]
+url = ../../..
```

Comment:
In the main project right now we ask end users to run `meson wrap install <lib>`, but that doesn't make sense to do with this nanoarrow.wrap, so I ended up putting all the wrap files here

Maybe even in the main project we should just add the wrap files? That would technically pin a version for third party dependencies (if not installed elsewhere on the system) which may or may not be desired